README.md

```diff
@@ -15,20 +15,38 @@ Use the component's HTML tag wherever you want:
 <myuw-profile
     login-url=""
     logout-url=""
-    session-endpoint=""
     background-color=""
-    open-right
 >
 </myuw-profile>
 ```
 
+### Login event
+
+To tell the component that there is an active session, dispatch a CustomEvent called "myuw-login":
+
+```js
+var customEvent = new CustomEvent('myuw-login', {
+  detail: { // required object "detail"
+    person: { // required object "person"
+      "firstName": "User" // required property "firstName"
+    }
+  }
+});
```

Comment:
@thevoiceofzeke In order for the custom events to bubble up through the DOM a `bubbles` property has to be explicitly set: https://developer.mozilla.org/en-US/docs/Web/Guide/Events/Creating_and_triggering_events#Event_bubbling

Reply:
Is this something you think we should include? If the event is dispatch and listener are both on `document` (or on the same element), we don't need the event to bubble. Is there a reason someone would want to dispatch the event from a narrower scope?